examples/basic/versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "4.30.0"
+      version = ">= 4.40"
```

why not 
```suggestion
      version = ">= 4.30"
```
Like we've had it everywhere else since basically ever?